sc/social/like/browser/imagefallback.py

```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8 -*-
+from plone import api
+from plone.formwidget.namedfile.converter import b64decode_file
+from plone.namedfile.browser import Download
+from plone.namedfile.file import NamedImage
+from sc.social.like.interfaces import ISocialLikeSettings
+
+
+class ImageFallBack(Download):
+
```

Comment:
please add a docstring expaining the purpose of this helper view.

Reply:
ok